app/assets/javascripts/controllers/middleware_datasource/middleware_datasource_controller.js

```diff
@@ -1,55 +1,69 @@
 ManageIQ.angular.app.controller('mwAddDatasourceController', MwAddDatasourceCtrl);
+'use strict';
```

Comment:
This doesn't do anything, `use strict` must be the fist statement in a function. (Which is lucky, because if it did, it would break javascript that gets included after, since we concatenate them all.)